index.html

```diff
@@ -441,6 +442,37 @@ <h3>Integrity Validation</h3>
           <p class="issue" data-number="143">
             Provide examples for Data Integrity and VC-JWT Credential Schemas
           </p>
+          <p>
+            Credential Schemas of type <a href="#jsonschema2023">JsonSchema2023</a> MAY
+            be annotated with integrity information by adding the property `digestSRI` to the `credentialSchema` value
+            in the Verifiable Credential which contains the schema, as described in <a data-cite="vc-data-model/#integrity-of-related-resources">Integrity of Related Resources</a>.
```

Comment:
```suggestion
            be annotated with integrity information by adding the `digestSRI` property to the `credentialSchema` value
            in the Verifiable Credential which contains the schema, as described in
            <a data-cite="vc-data-model/#integrity-of-related-resources">Integrity of Related Resources</a>.
```

Reply:
Done in https://github.com/w3c/vc-json-schema/pull/177/commits/2a4bd83e18bed0a75acc0c6f63a2f335a6674a30